provisioning-workflows/zOS/NetworkResources/Provision_Network_Resource.xml

```diff
@@ -0,0 +1,252 @@
+    <workflowInfo>
+        <parallelSteps>true</parallelSteps>
+        <workflowID scope="none">Provision_Network_Resource</workflowID>
+        <workflowDescription>Provision Network Resources</workflowDescription>
+        <workflowVersion>1.0</workflowVersion>
+        <vendor>IBM</vendor>
+        <Provisioning>
+            <productID>zOSMF</productID>
+            <productName>z/OS</productName>
+            <productVersion>Version 2.4</productVersion>
+            <softwareType>zOS</softwareType>
+        </Provisioning>
+    </workflowInfo>
+    <variable name="OBTAIN_DVIPA" visibility="public">
+        <label>OBTAIN_DVIPA</label>
+        <abstract>Obtain DVIPA flag</abstract>
+        <description>Flag to indicate whether or not to obtain DVIPA resource</description>
+        <category>Boolean Variables</category>
+        <string valueMustBeChoice="true">
+            <choice>YES</choice>
+            <choice>NO</choice>
+        </string>
+    </variable>
+    <variable name="OBTAIN_PORT" visibility="public">
+        <label>OBTAIN_PORT</label>
+        <abstract>Obtain Port flag</abstract>
+        <description>Flag to indicate whether or not to obtain port resource</description>
+        <category>Boolean Variables</category>
+        <string valueMustBeChoice="true">
+            <choice>YES</choice>
+            <choice>NO</choice>
+        </string>
+    </variable>
+    <variable name="OBTAIN_APPLID" visibility="public">
+        <label>OBTAIN_APPLID</label>
+        <abstract>Obtain APPLID flag</abstract>
+        <description>Flag to indicate whether or not to obtain APPLID resource</description>
+        <category>Boolean Variables</category>
+        <string valueMustBeChoice="true">
+            <choice>YES</choice>
+            <choice>NO</choice>
+        </string>
+    </variable>
+    <variable name="IP_ADDRESS" visibility="public">
+        <label>IP_ADDRESS</label>
+        <abstract>TCP/IP DVIPA IP Address</abstract>
+        <description>TCP/IP DVIPA IP Address</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="IP_ADDRESS_ID" visibility="private">
+        <label>IP_ADDRESS_ID</label>
+        <abstract>Object ID of the IP Address resource</abstract>
+        <description>The object id of the allocated IP address to use during deprovisioning</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="PORT" visibility="public">
+        <label>PORT</label>
+        <abstract>TCP/IP Port Number</abstract>
+        <description>TCP/IP port number</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="PORT_ID" visibility="private">
+        <label>PORT_ID</label>
+        <abstract>Object ID of the port resource</abstract>
+        <description>The object id of the reserved Port address to use during deprovisioning</description>
```

Comment:
```suggestion
        <description>The object ID of the reserved Port Address to use during provisioning</description>
```

Reply:
Or maybe instead of these suggestions, use the descriptions you provided in the README for these variables

Reply:
Done